apps/dashboard/src/components/buttons/TransactionButton.tsx

```diff
@@ -119,6 +119,7 @@ export const TransactionButton: React.FC<TransactionButtonProps> = ({
                 : `This action will trigger ${transactionCount} ${transactionCount > 1 ? "transactions" : "transaction"}`
             }
           >
+            {/* to be completed */}
```

Comment:
also might need help with this

Reply:
Let's just ignore this for now, We'll remove this when refactoring TransactionButton